dotnet/src/Planners/Planners.Core/Planners.Core.csproj

```diff
@@ -0,0 +1,68 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <!-- THIS PROPERTY GROUP MUST COME FIRST -->
+    <AssemblyName>Microsoft.SemanticKernel.Planners.Core</AssemblyName>
+    <RootNamespace>Microsoft.SemanticKernel.Planners</RootNamespace>
+    <TargetFramework>netstandard2.0</TargetFramework>
+  </PropertyGroup>
+
+  <Import Project="$(RepoRoot)/dotnet/nuget/nuget-package.props" />
+  <Import Project="$(RepoRoot)/dotnet/src/InternalUtilities/src/InternalUtilities.props" />
+
+  <PropertyGroup>
+    <!-- NuGet Package Settings -->
+    <Title>Semantic Kernel - Planners</Title>
+    <Description>Semantic Kernel Planners</Description>
```

Comment:
Should we advertise the types of planners included here?

Reply:
Good idea, fixed.